src/cheri/app-cheri-instructions.adoc

```diff
@@ -84,3 +84,43 @@ include::insns/zcmt_cmjalt.adoc[]
 include::insns/zcmt_cmjt.adoc[]
 
 endif::[]
+
+ifdef::cheri_standalone_spec[]
+
+== ISA changes since 0.9.5
+
+Many mnemonics have been renamed since v0.9.5 of the specification a shown in <<ISA_renames_095>>.
+
+.ISA renames since 0.9.5
+[#ISA_renames_095]
+[width="100%",options=header,cols="2,2,2",]
+|==============================================================================
+include::generated/old_and_new_mnemonics_table_body.adoc[]
+|==============================================================================
+
+Some instructions have been added as shown in <<ISA_added_095>>.
+
+.Instructions added since 0.9.5
+[#ISA_added_095]
+[width="100%",options=header,cols="2,2,2",]
+|==============================================================================
+include::generated/new_instructions_table_body.adoc[]
```

Comment:
Does this now include YSEAL/YUNSEAL? they aren't part of the ratified package so should probably omit it. Also this table would probably be better after the text? Otherwise it seems like there is more change than there actually was?